docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,261 @@
+k8ssandra-reaper-operator-858cd89bdd-7gfjj            1/1     Running     0          6m36s
+k8ssandra-reaper-schema-4gshj                         0/1     Completed   0          3m3s
+prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running     1          6m32s
+```
+
+## Create some data and back it up
+
+Extract the username and password to access Cassandra (the password is different for each installation unless it is explicitly set at install time) into variables:
+```
+% username=$(kubectl get secret k8ssandra-superuser -n k8ssandra -o jsonpath="{.data.username}" | base64 --decode)
+% password=$(kubectl get secret k8ssandra-superuser -n k8ssandra -o jsonpath="{.data.password}" | base64 --decode)
+```
+Connect through CQLSH on one of the nodes:
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u $username -p $password
+```
+Copy/paste the following statements into the CQLSH prompt and press enter:
+```
+CREATE KEYSPACE medusa_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+USE medusa_test;
+CREATE TABLE users (email TEXT PRIMARY KEY, name TEXT, state TEXT);
+INSERT INTO users (email, name, state) VALUES ('alice@example.com', 'Alice Smith', 'TX');
+INSERT INTO users (email, name, state) VALUES ('bob@example.com', 'Bob Jones', 'VA');
+INSERT INTO users (email, name, state) VALUES ('carol@example.com', 'Carol Jackson', 'CA');
+INSERT INTO users (email, name, state) VALUES ('david@example.com', 'David Yang', 'NV');
+```
+Check that the rows were properly inserted:
+```
+SELECT * FROM medusa_test.users;
+
+ email             | name          | state
+-------------------+---------------+-------
+ alice@example.com |   Alice Smith |    TX
+   bob@example.com |     Bob Jones |    VA
+ david@example.com |    David Yang |    NV
+ carol@example.com | Carol Jackson |    CA
+
+(4 rows)
+```
+
+Now backup this data, and check that files get created in your MinIO bucket. 
+
+To that end, use the following command:
+
+```
+helm install my-backup k8ssandra/backup -n k8ssandra --set name=backup1,cassandraDatacenter.name=dc1
+```
+
+Since the backup operation is asynchronous, you can monitor its completion by running the following command:
+
+```
+kubectl get cassandrabackup backup1 -n k8ssandra -o jsonpath={.status.finishTime}
+```
+
+As long as this doesn’t output a date and time, then the backup is still running. With the amount of data present and the fact that you’re using a locally accessible backend, this should complete quickly.
+
+Now refresh the MinIO UI and you should see some files in the k8ssandra-medusa bucket:
+
+![K8ssandra Backup Files](k8ssandra-medusa-backup.png)
+
+An index folder should appear (it is Medusa’s backup index) and then another folder that is specific to each Cassandra node in the cluster (in this case there is only one node).
+
+## Deleting the data and restoring the backup
+
+Delete rows one by one, and check that the table is empty:
```

Comment:
Is there a reason why we don't use `TRUNCATE`?

```suggestion
`TRUNCATE` the table and verify it is empty:
```

Reply:
@bradfordcp, we had a discussion around avoiding truncate because it raises the question of restoring from the auto snapshot, and we didn't want to go down that path. See [this comment](https://github.com/k8ssandra/k8ssandra/pull/605#pullrequestreview-626870590).